Utils/Dataflow/095_datasetInfoAMI/update_sample

```diff
@@ -4,6 +4,7 @@ base_dir=$(readlink -f $(dirname "$0"))
 
 sample=$base_dir/output/datasets.ndjson
 
+git checkout $sample
 mv $sample{,.bak}
```

Comment:
@mgolosova 
As I remember, "git checkout" allows to switch to a remote existed branch. Can't figure out how it will work in that case.

Reply:
Those strings were removed in d0af9f0, but I would like to ask anyway what was the main idea of c0b628e.

Reply:
There are a lot of ways to use `git checkout`: https://git-scm.com/docs/git-checkout.
[One of them](https://git-scm.com/docs/git-checkout#git-checkout-emgitcheckoutemlttree-ishgt--ltpathspecgt82308203) allows to update files in the working tree:
```
git checkout [<tree-ish>] [--] <pathspec>…​
    Overwrite paths in the working tree by replacing with the contents
    in the index or in the <tree-ish> (most often a commit). When a <tree-ish>
    is given, the paths that match the <pathspec> are updated both in the index
    and in the working tree.
```